pkg/network/filter/packet_source.go

```diff
@@ -0,0 +1,35 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
```

Comment:
you have `packet_source.go` with `go:build linux` and you have `packet_source_linux.go` (which also has `go:build linux`, but the linux specific file is implied from the name of the file)
what is in the difference?

Reply:
I need to remove this build tag.